tgui/packages/tgui/interfaces/Safe.js

```diff
@@ -0,0 +1,119 @@
+import { useBackend } from "../backend";
+import { Window } from "../layouts";
+import { Box, Button, Flex, Section } from "../components";
+const safeDialSrc = require('../assets/safe_dial.png');
+
+export const SafeContents = (properties, context) => {
+  const { data, act } = useBackend(context);
+  const {
+    contents,
+  } = data;
+
+  const close = () => {
+    act('open');
+  };
+
+  return (
+    <Section>
+      <Button content="Close" onClick={close} />
```

Comment:
But the button inside the `<Section buttons={}` attribute, and also add a section title